### DISCUSSION TITLE
Array should be contains defined values Constraints.

### DISCUSSION THREAD

**Discussion by qaasadi** [yalmip/YALMIP#1431] [Optimization modelling]
Hi, as a constraint i want my array be contains all number of 1-3. so i code this:
x=intvar(1,6,'full');
Constraints=[Constraints, ismember([1:3],x)];

but after running i faced with this error:
One of the constraints evaluates to a FALSE LOGICAL variable

please help me if you can.

**Comment by johanlofberg** [comment]
ismember(variable,possiblevalues)
